lighthouse-core/audits/seo/font-size.js

```diff
@@ -322,16 +320,9 @@ class FontSize extends Audit {
     let explanation;
     if (!passed) {
       const percentageOfFailingText = (100 - percentageOfPassingText) / 100;
-
-      // if we were unable to visit all text nodes we should disclose that information
-      if (visitedTextLength < totalTextLength) {
-        const percentageOfVisitedText = (visitedTextLength / totalTextLength);
-        explanation = str_(UIStrings.explanationWithDisclaimer,
-          {
-            decimalProportion: percentageOfFailingText,
-            decimalProportionVisited: percentageOfVisitedText,
-          });
-      }
+      explanation = str_(UIStrings.explanation, {
```

Comment:
should we just remove this?

looks like this

![image](https://user-images.githubusercontent.com/4071474/72024342-04df9c00-322a-11ea-92db-c12f2f535e46.png)